spec/client/builds_spec.rb

```diff
@@ -50,6 +50,17 @@
     end
   end
 
+  context '#paginated_build' do
+    it 'returns the build' do
+      VCR.use_cassette 'build' do
+        client.auto_paginate = true
```

Comment:
Just make sure that client is instantiated new for every spec. Otherwise you'll leak that state.

Reply:
Yup I verified that it is.